### PR TITLE
refactored document generation to use widdershins

### PR DESCRIPTION
**Pull request by rbell13**
We wanted a consistent way to generate easy to read documentation for these API definitions. Originally we began this work effort with looking to redocly. Widdershins seemed to generate better markdown and was easy to use for this end. We decided to use that to refactor the inconsistent methods to a singular method for keeping documentation up to date. This also supports a future state of a build pipeline that keeps the readmes up to date on code changes. 

This PR has a lot of changed files, but it really is updating all generation methods to use this one tool, the content of the APIs is unchanged

### CODE REVIEW

**Comment by rbell13** [comment]
@ysirotin image-analysis has been removed here as I do not believe it is one of our current core supported APIs. If this needs to remain. I can revert its removal. I was hoping to pare down to supported APIs

File: apis/image-analysis/api/readme.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+
+#### Enumerated Values
+
+|Property|Value|
+|---|---|
+|AlgorithmType|Face|
+|AlgorithmType|Finger|
+|AlgorithmType|Iris|
+|ImageDataset|NIST_MEDS|
+|ImageDataset|MDTF_RALLY2_ENROLLMENT|
```
[the remark this answers is not in the document]
Comment (rbell13) [reply apis/image-analysis/api/readme.md:305]: I am making the assumption this means to remove the enum value MDTF_RALLY2_ENROLLMENT. If this was incorrect, please let me know.

File: apis/match-to-id/api/readme.md [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+|» 1:1e4|string|true|none|none|
+|» 1:1e5|string|true|none|none|
+|» 1:1e6|string|true|none|none|
+
+#### Enumerated Values
+
+|Property|Value|
+|---|---|
+|AlgorithmModality|Face|
+|Test|MDTF_RIVTD_TRACK2|
```
[the remark this answers is not in the document]
Comment (rbell13) [reply apis/match-to-id/api/readme.md:528]: To clarify this statement, do you mean for me to add an enum value for Test that is "Test"? Does this mean to add a particular test value?

File: apis/matching-system/api/readme.md [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+<h1 id="the-maryland-test-facility-matching-system-interface-biometric-operations">Biometric Operations</h1>
+
+## Generate a template from the provided biometric image.
+
+<a id="opIdGenerateTemplate"></a>
+
+> Code samples
+
+```shell
+# You can also use wget
```
[the remark this answers is not in the document]
Comment (rbell13) [reply apis/matching-system/api/readme.md:43]: The readmes are all generated, this is an artifact of generating the shell examples, I do not believe it worth engineering a way to remove this and I think the value of shell examples is worth it

File: apis/image-analysis/api/readme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ysirotin) [reply apis/image-analysis/api/readme.md]: I think old events are certainly deprecated.  I think this field needs to be a string and not an enum as the enum would require us to continue supporting old events.  We could have requirements on a string for a specific event.

[the remark this answers is not in the document]
Comment (rbell13) [reply apis/image-analysis/api/readme.md]: I think we can do better now about handling images in general, and thing instead we can have these be specified in the submission process for labeling containers. I think it is worth considering removal. In the short term, and as this PR pertains to documentation generation tools, maybe we consider these as part of a near term future PR for these APIs. This PR is not for changing the design in a meaningful way, just docs

[the remark this answers is not in the document]
Comment (rbell13) [reply apis/image-analysis/api/readme.md]: I think this also means that the other comments can be considered outdated. I think we are ready to merge this. I have a ticket to discuss these elements. https://idslabs.atlassian.net/browse/SW-4532

**Review by johnarleyburns — APPROVED** [review]

**Pull request merged**